OPHD/StructureManager.cpp

```diff
@@ -248,6 +249,7 @@ void StructureManager::removeStructure(Structure* structure)
 	SKey s = SKey(structure);
 	for (auto& [componentTypeID, table] : mComponents)
 	{
+		UNUSED(componentTypeID);
```

Comment:
I would probably avoid using structured bindings here just to avoid using the `UNUSED` macro.